dace/codegen/targets/fpga.py

```diff
@@ -676,6 +851,162 @@ def deallocate_array(self, sdfg, dfg, state_id, node, function_stream,
                          callsite_stream):
         pass  # Handled by destructor
 
+    def compute_kernels(self, state: dace.SDFGState, default_kernel: int = 0):
```

Comment:
Maybe `partition_compute_kernels` or `annotate_compute_kernels`? Functions should preferably always start with a verb

Reply:
isn't compute a verb?

Reply:
I thought it was part of the noun "the compute kernels"?